ui/test/nuxeo-image-viewer.test.js

```diff
@@ -0,0 +1,326 @@
+        });
+
+        test('Should display zoom out option when looking to the first button', () => {
+          const zoomOutOption = toolbar.children[0];
+          expect(isElementVisible(zoomOutOption)).to.be.true;
+          expect(zoomOutOption.getAttribute('data-action')).to.equal('zoom-out');
+          expect(zoomOutOption.icon).to.equal('zoom-out');
+        });
+
+        test('Should display "fit to real size" option when looking to the second button (with zoom applied)', () => {
+          const fitToRealSizeOption = toolbar.children[1];
+          expect(isElementVisible(fitToRealSizeOption)).to.be.true;
+          expect(fitToRealSizeOption.getAttribute('data-action')).to.equal('fit-to-real-size');
+          expect(fitToRealSizeOption.icon).to.equal('nuxeo:fit-to-real-size');
+        });
+
+        test('Should display "fit to viewer option" when looking to the second button (without zoom applied)', () => {
+          viewer._fitToRealSize = true;
+
+          const fitToViewerOption = toolbar.children[1];
+          expect(isElementVisible(fitToViewerOption)).to.be.true;
+          expect(fitToViewerOption.getAttribute('data-action')).to.equal('fit-to-viewer');
+          expect(fitToViewerOption.icon).to.equal('nuxeo:fit-to-viewer');
+        });
+
+        test('Should display zoom out option when looking to the third button', () => {
+          const zoomInOption = toolbar.children[2];
+          expect(isElementVisible(zoomInOption)).to.be.true;
+          expect(zoomInOption.getAttribute('data-action')).to.equal('zoom-in');
+          expect(zoomInOption.icon).to.equal('zoom-in');
+        });
+
+        test('Should display zoom out option when looking to the fourth button', () => {
+          const rotateLeftOption = toolbar.children[3];
+          expect(isElementVisible(rotateLeftOption)).to.be.true;
+          expect(rotateLeftOption.getAttribute('data-action')).to.equal('rotate-left');
+          expect(rotateLeftOption.icon).to.equal('image:rotate-left');
+        });
+
+        test('Should display zoom out option when looking to the fifth button', () => {
+          const rotateRightOption = toolbar.children[4];
+          expect(isElementVisible(rotateRightOption)).to.be.true;
+          expect(rotateRightOption.getAttribute('data-action')).to.equal('rotate-right');
+          expect(rotateRightOption.icon).to.equal('image:rotate-right');
+        });
+      });
+    });
+  });
+
+  suite('Interactions', () => {
+    suite('Controls Toolbar', () => {
+      let viewer;
+      let toolbar;
+      setup(async () => {
+        viewer = await fixture(
+          html`
+            <nuxeo-image-viewer src="${base}/resources/sample.png" controls></nuxeo-image-viewer>
+          `,
+        );
+        await viewerLoad(viewer);
+        expect(isToolbarVisible(viewer)).to.be.true;
+        toolbar = viewer.$$('#toolbar');
+      });
+
+      test('Should call method to zoom out when first button is used', () => {
```

Comment:
Perhaps "clicked" would be more clear than "used"?
```suggestion
      test('Should call method to zoom out when first button is clicked', () => {
```
(applies to the tests below)

Reply:
Thanks!
Will use `is pressed` since we already have that pattern in other tests.